tasks/config/licenses.js

```diff
@@ -32,6 +32,7 @@ module.exports = function (grunt) {
         'UNLICENSE'
       ],
       overrides: {
+        'abbrev@1.0.7': ['MIT'],
```

Comment:
It's based on the package having a license in an undiscoverable format. Like, if it's mentioned in the README somewhere, or in a github issue, the license discovery tool won't be able to detect it automatically so we hard code it here